### DISCUSSION TITLE
Shulkers in Inventory

### DISCUSSION THREAD

**Discussion by NotEvenAUnicorn** [PurpurMC/Purpur#609] [__Dismissed__] [answered]
Adding a permission to be able to open shulkers from your inventory rather than just placing them...and if at all possible the permission be toggleable.

**Comment by CenCent** [comment]
They could also add a feature where items automatically go into shulkers if that item was already inside. If there is empty space then it wouldn't fill that empty space. For example, you are digging dirt and there is dirt in a shulker then the collected dirt automatically goes in the shulker but if there is empty space in the shulker then that space stays empty and the rest of the dirt goes into your inventory.

**Comment by katekerllenevich** [comment, marked as answer]
These are better off in a plugin
